### PR TITLE
fix(plugin/tcp-log):repeated sslhandshake in [tcp-log] plugin

### PR DESCRIPTION
**Pull request by chenyyyang**
### Summary

FIX: [Repeated sslhandshake in [tcp-log] plugin](https://github.com/Kong/kong/issues/11052)

### Checklist

- [ √] The Pull Request has tests
- [x] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/README.md)
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

change the code in  tcp-log/handler.lua

### Issue reference

Fix #[_[11052]_](https://github.com/Kong/kong/issues/11052)

### CODE REVIEW

**Comment by tzssangglass** [comment]
hi @chenyyyang , we need a changelog(you can learn from other PRs), and if you can add test cases to cover this fix, it maybe better!

**Comment by chenyyyang** [comment]
I have already browsed the other PRs but no changelog could be found.Could you provide a template?

**Comment by tzssangglass** [comment]
> I have already browsed the other PRs but no changelog could be found.Could you provide a template?

https://github.com/Kong/kong/pull/11786/files#diff-1c9ba4aa7ac45698a290b3506fc00dc16289cd098483e8e7e2bb580d0b40a8c2

**Comment by bungle** [comment]
@chenyyyang, looks good! Just add the changelog. See: https://github.com/Kong/kong/blob/master/changelog/changelog-template.yaml

Add change-log here:
https://github.com/Kong/kong/tree/master/changelog/unreleased/kong

Give it a name: `fix-tcp-log-sslhandshake.yml`

**Comment by chenyyyang** [comment]
Sure.I have added the changelog.

**Review by bungle — APPROVED** [review]

File: changelog/unreleased/kong/fix-tcp-log-sslhandshake.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+message: fix an issue that repeated ssl handshake in tcp-log plugin
```
Comment (tzssangglass) [inline changelog/unreleased/kong/fix-tcp-log-sslhandshake.yml:1]:
```suggestion
message: **tcp-log**: fix an issue that repeated ssl handshake
```

**Comment by chenyyyang** [comment]
Done @tzssangglass

**Review by tzssangglass — APPROVED** [review]

File: changelog/unreleased/kong/fix-tcp-log-sslhandshake.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+message: **tcp-log**: fix an issue that repeated ssl handshake
```
Comment (tzssangglass) [inline changelog/unreleased/kong/fix-tcp-log-sslhandshake.yml:1]:
```suggestion
message: "**tcp-log**: fix an issue that repeated ssl handshake"
```

previously, I mislead you. It should have double quotation marks added 😅

**Review by tzssangglass — CHANGES_REQUESTED** [review]

**Review by chenyyyang — COMMENTED** [review]
done？

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**